tests/integration/test_scmsteprun_view.py

```diff
@@ -158,3 +158,51 @@ def test_create(self, client, logged_in_user, scm_pipeline_run, scm_step_run):
         assert response.status_code == 201
         assert models.SCMStepRun.objects.filter(name='Release product').exists()
         assert models.SCMStepRun.objects.count() == initial_count + 1
+
+    def test_status_update_with_boolean(self, client, logged_in_user, scm_step_run):
+        assert scm_step_run.status == 'not started'
+
+        url = f'/scm-step-runs/{scm_step_run.public_identifier}/status_success/'
```

Comment:
Should the URL not be `../status-success/`?

Reply:
updated